db/migrations/20170915150351_drop_title_from_movies.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('title');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN name SET NOT NULL');
```

Comment:
As a heads up, these `ALTER` commands can be dangerous on large tables. They should be fine here, but when you get a chance, I recommend reading the [Writing Migrations](https://www.notion.so/lob/Writing-Migrations-15caf9dd14f74c47b498fd7bee8a7b51) doc. It explains how we usually split up constraint creation such that it minimizes the locks on the database.